netbox/project-static/src/appliedFilter.ts

```diff
@@ -0,0 +1,18 @@
+import { isTruthy } from './util';
+
+/**
+ * Initialize the applied filter display.
+ *
+ */
+export function initAppliedFilter(): void {
+  const appliedFilter = document.getElementById('appliedfilters');
+  if (isTruthy(appliedFilter)) {
+    const divResults = document.getElementById('results');
+    if (isTruthy(divResults)) {
+      const savedFilterSelect = divResults.getElementsByTagName('select')[0];
+      if (savedFilterSelect.selectedOptions.length > 0) {
+        appliedFilter.hidden = true;
```

Comment:
We should not rely on Javascript to hide this element; rather it should simply not be rendered in the template.

@arthanson can you help Julio with this please?